### PR TITLE
Works and about section

### PR DESCRIPTION
**Pull request by DammyShittu**
In this pull request, the;
- About  section and,
- Work Section 
 were added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,6 +94,96 @@ <h2 class="work-1 first">Work</h2>
...
+        <li class="stack">Css</li>
+        <li class="stack">Html</li>
+        <li class="stack">Bootstrap</li>
+        <li>Ruby</li>
+       </ul>
+      </div>
+ 
+      <button>See Project</button>
+     </aside>
+    </div>
```
Comment (arslanbisharat) [inline index.html:123]: - [ ] The projects should be enclosed in `ul` and `li `tags.

File: index.html [hunk 2]
```diff
@@ -94,6 +94,96 @@ <h2 class="work-1 first">Work</h2>
...
+       <ul>
+        <li class="stack">Css</li>
+        <li class="stack">Html</li>
+        <li class="stack">Bootstrap</li>
+        <li>Ruby</li>
+       </ul>
+      </div>
+ 
+      <button>See Project</button>
+     </aside>
```
Comment (arslanbisharat) [inline index.html:142]: - [ ] The projects should be enclosed in `ul` and `li `tags.

File: index.html [hunk 3]
```diff
@@ -94,6 +94,96 @@ <h2 class="work-1 first">Work</h2>
...
+    </div>
```
Comment (arslanbisharat) [inline index.html:163]: - [ ] The projects should be enclosed in `ul` and `li `tags.

File: index.html [hunk 4]
```diff
@@ -94,6 +94,96 @@ <h2 class="work-1 first">Work</h2>
...
+   </div>
```
Comment (arslanbisharat) [inline index.html:184]: - [ ] The projects should be enclosed in `ul` and `li `tags.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @DammyShittu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

#### About myself

1. The container of the section is a `section` tag
2. The section has the proper background image (decorations are always background images, not `img` tags)
3. The title (e.g., "About myself") uses a header tag `hx`
4. The paragraph (e.g., "I'm a software ...") is a `p` tag
5. If there are other titles (e.g., "Let's connect") use a header tag `hx`
6. The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
7. The button (e.g., "Get my resume") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design)
8. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
9. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`
10. If the section has a navigation link (e.g., "Work") it uses a `a` tag


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by carlos-ssh — APPROVED** [review]
Hi @DammyShittu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**